tests/utils/utils.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios');
+
+const { MOCK_AUTH_SERVICE = "http://localhost:8001" } = process.env;
+
+async function generateTestToken() {
+    let token, tokenRequest;
+
+    try {
+        tokenRequest = await axios.get(MOCK_AUTH_SERVICE + '/oauth2/default/v1/token');
+        token = tokenRequest.data.token;
+    } catch (error) {
+        throw ({message: error?.message || "Unable to fetcha dev token."})
```

Comment:
let's also create a new error here instead of throwing an object directly :)

Reply:
Oh yes ! I forgot this one.